src/Neo.CLI/CLI/MainService.Vote.cs

```diff
@@ -258,5 +223,29 @@ private void OnGetAccountState(UInt160 address)
                 ConsoleHelper.Error("Error parsing the result");
             }
         }
+        /// <summary>
+        /// Get account or log a warm
+        /// </summary>
+        /// <param name="account"></param>
+        /// <returns>account or null</returns>
+        private WalletAccount? GetValidAccountOrWarn(UInt160 account)
+        {
+            var acct = CurrentWallet?.GetAccount(account);
+            if (acct == null)
+            {
+                ConsoleHelper.Warning("This address isn't in your wallet!");
+                return null;
+            }
+            if (acct.Lock || acct.WatchOnly)
+            {
+                ConsoleHelper.Warning("Locked or WatchOnly address.");
+                return null;
+            }
+            return acct;
+        }
+
+        private byte[] BuildNativeScript(string method, params object?[] args)
+            => NativeContract.NEO.Hash.MakeScript(method, args);
```

Comment:
```suggestion
        private byte[] BuildNeoScript(string method, params object?[] args)
            => NativeContract.NEO.Hash.MakeScript(method, args);
```
Please change the name from `BuildNativeScript` to `BuildNeoScript`, it's not good since we have multi native contract, but this seems is only for Neo contract.